layouts/registration.vue

```diff
@@ -1,7 +1,27 @@
 <template>
   <div>
-    <Header />
+    <SidebarMobile :show="sidebarOpen" @close-sidebar="clsoeSidebar" />
```

Comment:
same here

Reply:
Hello kak @Ibwedagama, thanks for the review. You can see the changes [here](https://github.com/jabardigitalservice/dedi-landing-page/pull/142/commits/04ed2527bd47999725456c67a9dabd4d44d2b95c)